paddle/fluid/framework/ir/squared_mat_sub_fuse_pass.cc

```diff
@@ -394,5 +394,5 @@ REGISTER_PASS_CAPABILITY(squared_mat_sub_fuse_pass)
             .EQ("square", 0)
             .LE("elementwise_mul", 1)
             .LE("elementwise_sub", 1)
-            .EQ("fill_constant", 1)
+            .LE("fill_constant", 2)
```

Comment:
Not important. Why change the file property `100644 → 100755`?

Reply:
done